templates/ocs-deployment/csi/cephfs/csi-nodeplugin-rbac_cephfs.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
```

Comment:
\-\-\-
as first line in document please in each yaml file

Can we be consistent and using dashes in the names of files?

Reply:
Done